.github/workflows/pushrelease.yml

```diff
@@ -154,22 +153,20 @@ jobs:
 
       - uses: r-lib/actions/setup-r@v2
 
-      - uses: r-lib/actions/setup-pandoc@v1
+      - uses: r-lib/actions/setup-pandoc@v2
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
           cache-version: 2
           extra-packages: |
+            any::messydates
```

Comment:
Is this good practice, to have itself as a dependency?

Reply:
This is done in packages where it is necessary to install the package to build the readme file; this is also the case for manydata for example. Otherwise, this portion of the workflow will fail. Please let me know if there is anything you would like me to do here.